### PR TITLE
add gm hermes cc flag

### PR DESCRIPTION
**Pull request by AlianBenabdallah**
Closes: [#2416](https://github.com/informalsystems/ibc-rs/issues/2416)

## Description

`gm hermes cc` prints the same list of commands with `--new-client-connection yes`
`gm hermes cc --exec` executes the output of gm hermes cc
______

### PR author checklist:
- [ ] Added changelog entry, using [`unclog`](https://github.com/informalsystems/unclog).
- [ ] Added tests: integration (for Hermes) or unit/mock tests (for modules). 
- [x] Linked to GitHub issue.
- [x] Updated code comments and documentation (e.g., `docs/`).

### Reviewer checklist:

- [ ] Reviewed `Files changed` in the GitHub PR explorer.
- [ ] Manually tested (in case integration/unit/mock tests are absent).

### CODE REVIEW

File: scripts/gm/README.md [hunk 1]
```diff
@@ -323,4 +325,6 @@ This will
 * generate the config for hermes
 * print the `create client` commands for a full-mesh connection among the IBC node networks.
 
-Pick and choose the connections from the list that you want to create.
+
+
+Pick and choose the connections from the list that you want to create or run `gm hermes cc --exec` to run all the commands.
```
Comment (ljoss17) [inline scripts/gm/README.md:330]:
```suggestion
Pick and choose the connections from the list that you want to create or run `gm hermes cc --exec` to run all the commands.
```
Nit-pick: unnecessary new lines

File: scripts/gm/README.md [hunk 2]
```diff
@@ -314,7 +315,8 @@ Run the below:
 gm start
 gm hermes config
 gm hermes keys
-gm hermes cc
+gm hermes cc 
+
```
Comment (ljoss17) [inline scripts/gm/README.md:319]: Nit-pick: unnecessary new line

**Review by ljoss17 — APPROVED** [review]
Small aesthetic changes. All else looks great. the minimal configuration example works and the `gm hermes cc` command with and without `--exec` works as expected

File: scripts/gm/README.md (hunk 2 above)
Comment (AlianBenabdallah) [reply scripts/gm/README.md]: Nice catch ! Fixed that

**Pull request merged**